### PR TITLE
Added Drexel University Picotte template.

### PR DESCRIPTION
**Pull request by erinvchen**
## Description
We added a template.

## Motivation and Context
We're using this for simulations on the Picotte cluster.

## Types of Changes

- [ ] Documentation update
- [ ] Bug fix
- [x] New feature
- [ ] Breaking change<sup>1</sup>

<sup>1</sup>The change breaks (or has the potential to break) existing functionality.

## Checklist:

- [x] I am familiar with the [**Contributing Guidelines**](https://github.com/glotzerlab/signac-flow/blob/master/CONTRIBUTING.md).
- [x] I agree with the terms of the [**Contributor Agreement**](https://github.com/glotzerlab/signac-flow/blob/master/ContributorAgreement.md).
- [ ] My name is on the [list of contributors](https://github.com/glotzerlab/signac-flow/blob/master/contributors.yaml).
- [ ] My code follows the [code style guideline](https://github.com/glotzerlab/signac-flow/blob/master/CONTRIBUTING.md#code-style) of this project.
- [ ] The changes introduced by this pull request are covered by existing or newly introduced tests.

If necessary:
- [ ] I have updated the API documentation as part of the package doc-strings.
- [ ] I have created a separate pull request to update the [framework documentation](https://docs.signac.io/) on [signac-docs](https://github.com/glotzerlab/signac-docs) and linked it here.
- [ ] I have updated the [changelog](https://github.com/glotzerlab/signac-flow/blob/master/changelog.txt).

### CODE REVIEW

**Comment by bdice** [comment]
I worked on this with @erinvchen during office hours. I'm planning to make a few small changes to add this to the signac-flow docs.

**Comment by bdice** [comment]
@erinvchen Could you add yourself to the [list of contributors](https://github.com/glotzerlab/signac-flow/blob/master/contributors.yaml)? That's the last step needed for this PR.

@glotzerlab/signac-committers This is ready for review. Since I helped write this, could I ask another committer to review this template? @erinvchen has verified that it works on the Picotte cluster for CPU and GPU submissions.

**Review by bdice — APPROVED** [review]
This needs a second review but I think it is ready to merge, aside from adding @erinvchen to the [contributors list](https://github.com/glotzerlab/signac-flow/blob/master/contributors.yaml).

**Comment by csadorf** [comment]
> The link to the cluster documentation is not publicly accessible - I think it may require Drexel network access. However, this is the canonical source for cluster information (@erinvchen screen-shared with me while we wrote the template). I think it's fine to leave the link as-is.

@bdice Maybe add a quick note about this wherever the URL appears just so that users and future developers are not confused.

File: doc/supported_environments/drexel.rst
Comment (bdice) [inline doc/supported_environments/drexel.rst:6]:
```suggestion
`Link to official documentation (may require access to the Drexel University network) <https://proteusmaster.urcf.drexel.edu/urcfwiki/>`__
```

File: flow/environments/drexel.py
Comment (bdice) [inline flow/environments/drexel.py:11]:
```suggestion
    https://proteusmaster.urcf.drexel.edu/urcfwiki/ (this link may require access to the Drexel University network)
```

**Comment by bdice** [comment]
@erinvchen Can you add yourself to the [contributors list](https://github.com/glotzerlab/signac-flow/blob/master/contributors.yaml)? Then we can merge and release this PR.

File: flow/templates/drexel-picotte.sh
Comment (b-butler) [inline flow/templates/drexel-picotte.sh:15]: Does this correctly handle multi node GPU submissions?

**Review by b-butler — CHANGES_REQUESTED** [review]
I think when multiple GPU nodes are requested this will incorrectly assign the tasks per node.

File: flow/templates/drexel-picotte.sh
Comment (bdice) [reply flow/templates/drexel-picotte.sh:15]: @b-butler Perhaps — are you thinking that we are missing “divide tasks by the computed number of nodes and round up”? If you see something else, please clarify.

I believe this logic is copied from other templates so we should check carefully.

Comment (b-butler) [reply flow/templates/drexel-picotte.sh:15]: Yes, we do this below for the `dft` partition. We do this in GreatLake's template, but I think we also error there. However, we don't deal with multi node submissions there since we don't have enough resource assess for that. An example of the other is the Expanse template.

**Comment by b-butler** [comment]
@erinvchen will you have time to implement these changes.

File: flow/templates/drexel-picotte.sh
Comment (bdice) [reply flow/templates/drexel-picotte.sh:15]: This will be handled separately in #566.

**Review by b-butler — APPROVED** [review]
:+1:

**Comment by bdice** [comment]
@erinvchen We're going to merge this once tests pass - feel free to add yourself to the contributors list later, if you wish.

**Pull request merged**
